.github/workflows/labeler.yml

```diff
@@ -0,0 +1,25 @@
+name: "Label PR"
+
+on:
+  pull_request_target:
+    types: [edited, opened, synchronize, reopened]
```

Comment:
I think this line is not necessary. By default this event is run on `opened`, `synchronize`, and `reopened`. 
Nixpkgs adds `edited` to cover when a PR changes its base branch, which is very unlikely in home-manager
```suggestion
```